easybuild/easyconfigs/p/papermill/papermill-2.3.4-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'PythonBundle'
+
+name = 'papermill'
+version = '2.3.4'
+
+homepage = 'https://papermill.readthedocs.io'
+description = """Papermill is a tool for parameterizing and executing Jupyter Notebooks."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('IPython', '7.18.1'),  # nbclient, (nbformat<5.1.2)
+    ('PyYAML', '5.3.1'),    # pyyaml
+    ('tqdm', '4.60.0'),     # tqdm
+]
+
+builddependencies = [
+    ('binutils', '2.35'),
+]
+
+use_pip = True
+
+sanity_pip_check = True
+
+exts_list = [
+    ('nbformat', '5.1.3', {
+        'checksums': ['b516788ad70771c6250977c1374fcca6edebe6126fd2adb5a69aa5c2356fd1c8'],
+    }),
+    ('ansiwrap', '0.8.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/A/ansiwrap/'],
+        'sources': ['ansiwrap-0.8.4.zip'],
+        'checksums': ['ca0c740734cde59bf919f8ff2c386f74f9a369818cdc60efe94893d01ea8d9b7'],
+    }),
+    ('tenacity', '8.0.1', {
+        'checksums': ['43242a20e3e73291a28bcbcacfd6e000b02d3857a9a9fff56b297a27afdc932f'],
+    }),
+    ('textwrap3', '0.9.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/T/textwrap3/'],
+        'sources': ['textwrap3-0.9.2.zip'],
+        'checksums': ['5008eeebdb236f6303dcd68f18b856d355f6197511d952ba74bc75e40e0c3414'],
+    }),
+    (name, version, {
+        'checksums': ['be12d2728989c0ae17b42fcb05b623500004e94b34f56bd153355ccebb84a59a'],
+    }),
+]
+
```

Comment:
Are there any `sanity_check_commands` you could add here?
E.g. `papermill --version`?

Reply:
Yes, that would work. Sorry I did not see the suggestion.